src/schemas/generateMegaRecord.ts

```diff
@@ -82,6 +82,12 @@ export async function generateMegaRecord(schema: any) {
             });
         }
 
+        // specify the proto field for the topic
+        if ("proto" in _meta) {
```

Comment:
I think this should be invalid if the user doesn’t specify proto output format. In other words, if they have proto in _meta but are using -f avro for output, then _meta.proto should be ignored.

Reply:
The problem with this is that I would have to pass in the `format` into the `generateMegaRecord` method to skip the `if("proto" in _meta)` check.

the `_meta.proto` is effectively ignored already by all output formats, including Avro, because `_meta.proto` is only referenced in the `protoFormat.ts`